hack/make.sh

```diff
@@ -250,6 +248,7 @@ test_env() {
 		DOCKER_REMAP_ROOT="$DOCKER_REMAP_ROOT" \
 		DOCKER_REMOTE_DAEMON="$DOCKER_REMOTE_DAEMON" \
 		GOPATH="$GOPATH" \
+		GOTRACEBACK=all \
```

Comment:
This was added so that the panics that go-check produces give a full goroutine dump, otherwise the trace is useless since it's just a trace of the main go-check code, not the timed-out test code.